importers/data_sources/README.md

```diff
@@ -0,0 +1,64 @@
+# Install data_sources dataset
+
+The script and data within this directory allow one to load the _data sources_ into the RE database. The default RE base url is `http://localhost:5000`, as defined in `importers/utils/config.py`, which can be redefined with the environment variable `RES_API_URL`. 
+
+This script has only been used in development thus far. In this case, the local RE Api container must be started via `make start-ui-dev`, as this starts the services with the `docker-compose.yaml` configuration located in `local-ui-dev`, which initializes all collections.
+
+Of course, in development you may have a different process; and in production I (Erik) have no clue how data is initialized or loaded. This script should work fine for initial load, and can be used for subsequent updates if the script is pointed at a data file which contains just the new documents.
```

Comment:
I must have removed the reference to that narrative, because it isn't in the file any longer. It was probably there as a aide-mémoire, but I don't remember now why.